crypto/src/value.rs

```diff
@@ -45,15 +45,9 @@ impl TryFrom<pb::Value> for Value {
     }
 }
 
-#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Default)]
```

Comment:
The existing `Default` implementation was identical to the derived implementation.